src/pytorch_lightning/README.md

```diff
@@ -130,24 +130,24 @@ conda install pytorch-lightning -c conda-forge
 
 the actual status of 1.6 \[stable\] is following:
 
-![CI basic testing](https://github.com/PyTorchLightning/pytorch-lightning/workflows/CI%20basic%20testing/badge.svg?branch=release%2F1.5.x&event=push)
-![CI complete testing](https://github.com/PyTorchLightning/pytorch-lightning/workflows/CI%20complete%20testing/badge.svg?branch=release%2F1.5.x&event=push)
-![PyTorch & Conda](https://github.com/PyTorchLightning/pytorch-lightning/workflows/PyTorch%20&%20Conda/badge.svg?branch=release%2F1.5.x&event=push)
-![TPU tests](https://github.com/PyTorchLightning/pytorch-lightning/workflows/TPU%20tests/badge.svg?branch=release%2F1.5.x&event=push)
-![Docs check](https://github.com/PyTorchLightning/pytorch-lightning/workflows/Docs%20check/badge.svg?branch=release%2F1.5.x&event=push)
+![CI basic testing](https://github.com/Lightning-AI/lightning/workflows/CI%20basic%20testing/badge.svg?branch=release%2F1.5.x&event=push)
+![CI complete testing](https://github.com/Lightning-AI/lightning/workflows/CI%20complete%20testing/badge.svg?branch=release%2F1.5.x&event=push)
+![PyTorch & Conda](https://github.com/Lightning-AI/lightning/workflows/PyTorch%20&%20Conda/badge.svg?branch=release%2F1.5.x&event=push)
+![TPU tests](https://github.com/Lightning-AI/lightning/workflows/TPU%20tests/badge.svg?branch=release%2F1.5.x&event=push)
+![Docs check](https://github.com/Lightning-AI/lightning/workflows/Docs%20check/badge.svg?branch=release%2F1.5.x&event=push)
```

Comment:
same here.

Reply:
yes, all reference to `release/1.5.x` and shall be master or `release/pytorch` but then the referred function need to be updated

Reply:
Updated, please check if it's correct :)

Reply:
I'll make sure to update all of the badges in #13917!